exec/java-exec/src/main/resources/rest/profile/list.ftl

```diff
@@ -64,6 +65,8 @@
             <td>
               <div style="height:100%;width:100%">${query.getState()}</div>          
             <td>
+              <div style="height:100%;width:100%">${(.now?long - query.getStartTime()) / 1000} sec</div>          
```

Comment:
Rounding? Should 999 ms be shown as 0 seconds or 1 second. Actually, given the short duration of some queries, should the time be shown down to the ms? That is either 1234 ms or 1.234 sec. Or, we could be fancy and show only three digits of precision: 1.23 seconds, 123,000 seconds. Further, do we want to shift units as the numbers increase: 234 ms, 23.4 seconds, 15 min 34 sec. 1 hr 23 min.?